src/reference/asciidoc/amqp.adoc

```diff
@@ -5231,6 +5231,8 @@ RepublishMessageRecoverer recoverer = new RepublishMessageRecoverer(amqpTemplate
 };
 ----
 
+Starting with version 2.1, an `ImmediateRequeueMessageRecoverer` is  added to throw an `ImmediateRequeueAmqpException` which notifies a listener container to requeue the current failed message.
+
 ===== Exception Classification for Retry
 
 Spring Retry has a great deal of flexibility for determining which exceptions can invoke retry.
```

Comment:
I think we should mention elsewhere in the docs (somewhere where we talk about `defaultRequeueRejected` ?) that this new exception can be thrown by a listener to override the behavior if the `defaultRequeueRejected` has been changed from its default (`true`) to `false`.

Reply:
Doh. Has meant that when developed this exception, but been side-tracked with other stuff...
Will add soon.

Thanks